### PR TITLE
Update logging.md

### PR DESCRIPTION
**Pull request by mwachs5**
Clean up some verbiage that is no longer accurate

### CODE REVIEW

File: share/doc/wake/logging.md [hunk 1]
```diff
@@ -76,6 +76,11 @@ These can be concatenated with commas and include the normal log levels, so to c
 ```
 wake --fd:3="foo,info" mywakecode 3>foo_and_info.txt
 ```
+### Log Headers
+
+Each logging stream can optionally emit with a log header. See the
+"Configuring Wake" section of the [README](https://github.com/sifive/wake/README.md?plain=1#L140)
```
Comment (mwachs5) [inline share/doc/wake/logging.md:82]:
```suggestion
"Configuring Wake" section of the [README](https://github.com/sifive/wake/blob/master/README.md#configuring-wake)
```

File: share/doc/wake/logging.md [hunk 2]
```diff
@@ -20,22 +20,22 @@ and a display format (color and intensity).
...
-| LogLevel "bar"    |     Default     | "bar"         |
+| wake `def`        |  log level    |
+|-------------------|---------------|
+| logDebug          | "debug"       |
+| logInfo           | "info"        |
+| logEcho           | "echo"        |
+| logInteractive    | "interactive" |
+| logReport         | "report"      |
+| logWarning        | "warning"     |
+| logError          |  "error"      |
```
Comment (V-FEXrt) [inline share/doc/wake/logging.md:35]: nit: can you realign the  left quote on `"error"` and `"null"`

**Review by V-FEXrt — APPROVED** [review]

**Review by JakeSiFive — APPROVED** [review]
Thanks Megan!

**Comment by JakeSiFive** [comment]
@mwachs5 did you ever get merge permissions from us?

File: share/doc/wake/logging.md (hunk 2 above)
Comment (mwachs5) [reply share/doc/wake/logging.md:35]: oops just saw this

**Pull request merged**